drivers/wifi/simplelink/simplelink_support.c

```diff
@@ -218,7 +218,7 @@ void SimpleLinkWlanEventHandler(SlWlanEvent_t *wlan_event)
 		memcpy(sl_conn.bssid, wlan_event->Data.Connect.Bssid,
 		       BSSID_LEN_MAX);
 
-		LOG_INF("\n[WLAN EVENT] STA Connected to the AP: %s, "
+		LOG_INF("\n\r[WLAN EVENT] STA Connected to the AP: %s, "
```

Comment:
I believe even when it was initially introduced, there was a comment that it's a bad thing, and it now pays off. So, it's another chance to remove that \n altogether, there's a simple rule to follow: one logging action - one log line produced.

If that doesn't work, then at lease make it \r\n, as the standard patter for it goes.

Reply:
> \r\n, as the standard patter for it goes.

What standard is that? We should not add these \r, that is totally pointless thing to do. See also #10399 

I also think that adding \n to logging strings is useless as the output might go to some backend where \n makes no sense.

Reply:
OK, will (finally) remove those \n 's.

Reply:
:+1: